templates/users/userstatus.json

```diff
@@ -1,5 +1,6 @@
 {
 {% if loggedin %}	"username": "{{ username }}",
+{% endif %}{% if customname %}	"customname": "{{ customname }}",
 {% endif %}{% if loggedin %}	"email": "{{ email }}",
 {% endif %}	"loggedin": {% if loggedin %}true{% else %}false
 {% endif %}{% if moderator %},
```

Comment:
What's `templates/users/userstatus.json`  for? Is it for determining what gets sent as response?

Reply:
just a thought - should we always include `customname` as property?

Reply:
wouldn't make a difference on the JS side though:
```
if (result.customname) {
 ...
}
```
this'll do the right thing for `customname` when it's undefined, false, null, or even `""`

Reply:
oh yea it was just thought as I'm not a API design expert 😉